srcpkgs/haproxy/files/haproxy/run

```diff
@@ -1,3 +1,5 @@
 #!/bin/sh
 
-exec haproxy -f /etc/haproxy/haproxy.cfg
+exec haproxy -W \
+  -p /var/run/haproxy.pid \
```

Comment:
I suppose the PID file is needed for this handling?

Reply:
It was only storing the PID of the master process, not any of the workers. Removed!